### ISSUE TITLE
Postgres not working on v1.1.5-beta.16 Docker containers

### ISSUE THREAD

**Issue by jdeppa** [morpheus65535/bazarr#2063]
**Describe the bug**
On linuxserver.io's v1.1.5-beta.16 image (I have tried this and beta.8), Bazarr will not start if set up with Postgres following the wiki. Once I added the Postgres information in `config.ini`, and restarted the container, it shows after the lsio header: 
```
Bazarr exited.
Bazarr starting...
Traceback (most recent call last):
  File "/app/bazarr/bin/bazarr/main.py", line 22, in <module>
    from init import *  # noqa E402
  File "/app/bazarr/bin/bazarr/init.py", line 229, in <module>
    from app.database import init_db, migrate_db  # noqa E402
  File "/app/bazarr/bin/bazarr/app/database.py", line 101, in <module>
    class TableEpisodes(BaseModel):
  File "/app/bazarr/bin/bazarr/../libs/peewee.py", line 6342, in __new__
    cls._meta.add_field(name, field)
  File "/app/bazarr/bin/bazarr/../libs/peewee.py", line 6126, in add_field
    field.bind(self.model, field_name, set_attribute)
  File "/app/bazarr/bin/bazarr/../libs/peewee.py", line 4878, in bind
    self._db_hook(model._meta.database)
  File "/app/bazarr/bin/bazarr/../libs/peewee.py", line 4870, in _db_hook
    self._constructor = database.get_binary_type()
  File "/app/bazarr/bin/bazarr/../libs/peewee.py", line 4017, in get_binary_type
    return psycopg2.Binary
AttributeError: 'NoneType' object has no attribute 'Binary'
Bazarr exited.
Bazarr starting...
```
and it keeps repeating as Bazarr keep trying to start but dies immediately after.

If I remove the postgres config and restart the container, then it'll start up normally. hotio's image seem to ignore postgres configs, the postgres server did not show any activity from the bazarr's user.

**To Reproduce**
1. Add postgres config to `config.ini` following the wiki.
2. Restart container
3. Check container logs

**Expected behavior**
Container and Bazarr to start normally and creating the tables in postgres.

**Software (please complete the following information):**
 - Bazarr: v1.1.5-beta.16
 - OS: Ubuntu 22.04 (Docker, amd64)

**Additional context**
The container can communicate with each other (tested via ping).
bazarr user and database created in postgres manually.

**Comment by morpheus65535** [comment]
@roxedus @thespad any ETA for psycopg2 python Module? Thanks!

**Comment by thespad** [comment]
Ideally if you can add a `postgres-requirements.txt` or similar (or however you want to manage it) for the dev branch (and stable when it's released) then we can use that to install and if you need to change/limit versions or add any additional deps (because you decide to support mariadb & mysql as well or whatever) then it'll get pulled in automatically.

**Comment by morpheus65535** [comment]
@thespad Done!

Will be part of tonight dev branch release (beta).

Thanks!

**Comment by thespad** [comment]
I'll knock up a PR for dev in the morning then.

**Comment by halali** [comment]
@jdeppa can you confirm its working ?

**Comment by jdeppa** [comment]
Just tried, but no, still not working. Tried with image `"build_version": "Linuxserver.io version:- v1.1.5-beta.18-ls477 Build-date:- 2023-02-18T07:52:24+01:00"`. Same log output as above.

**Comment by jdeppa** [comment]
If I install `postgresql` in the container with `apk add postgresql`, after the installation, it works. So I guess it's a missing dependency in the container image. @Roxedus @thespad

**Comment by halali** [comment]
ok but you cant install postgresql inside container... you need your own postgresql somewhere else than bazarr container...
there were missing only psycopg2 which is driver for python that helps comunicate with your postgresql database

**Comment by jdeppa** [comment]
Yes, I know that. I have a postgres server in another container with other database on it. 

I did a quick search of the error and got to [this](https://support.prodi.gy/t/postgresql-connection-error/809/9) page, and the last post suggested that `psycopg2` will need a library `libpq.5.dylib`, so I checked which package provides this library and found out that `postgresql` package does, so I tried installing the package in the container in order to supply the library to `psycopg2`.

The connection to the postgresql server that is in the `config.ini` is still my postgres server NOT in the container, don't worry.
